### PR TITLE
Update environment to include the pandas nightly build

### PR DESCRIPTION
**Pull request by kcpevey**

### CODE REVIEW

**Comment by kcpevey** [comment]
ci failure is from notebooks that we already know don't run (see next PR)

File: README.md [hunk 1]
```diff
@@ -52,3 +52,9 @@ This helps us keep the repo size down so its quick to download.
 ```
 pre-commit install
 ```
+
```
Comment (kcpevey) [inline README.md:55]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -52,3 +52,8 @@ This helps us keep the repo size down so its quick to download.
...
+pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple pandas
+
+
+
```
Comment (kcpevey) [inline README.md:59]:
```suggestion

```

**Review by MarcoGorelli — APPROVED** [review]
no objections

**Pull request merged**